aten/src/ATen/native/cpu/MaxPoolKernel.cpp

```diff
@@ -85,15 +85,22 @@ compute_internal(
   using iVec = vec::Vectorized<integer_t>;
   // Pass I: init out lane
   iVec index0_vec = iVec(id0 * input_height * input_width + ih0 * input_width + iw0);
-  Vec out_vec = Vec(-std::numeric_limits<scalar_t>::infinity());
```

Comment:
`std::numeric_limits<scalar_t>::infinity()` is defined to be `0` for integral types.